### ISSUE TITLE
upload prebuild graph-node container to docker hub 

### ISSUE THREAD

**Issue by jellegerbrandy** [daostack/subgraph#69]
It would make lif easier upstream if we have pre-built containers for the graph-node that work together with the container from the migration repo (cf. issue #68 )

**Comment by ben-kaufman** [comment]
Not sure I exactly understood you. You mean to upload the subgraph to docker hub which can be used as the migration is used instead of the ganache in docker-compose?

**Comment by jellegerbrandy** [comment]
Yes exactly! We should probably also include this in the travis setup, as described in issue #17
